### PR TITLE
Adding missing public docstrings Bio.pdb Issue 1203

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #...

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/PDB/AbstractPropertyMap.py [hunk 1]
```diff
@@ -9,13 +9,16 @@
...
+    """Define base class, map holder of residue properties."""
+
     def __init__(self, property_dict, property_keys, property_list):
         """Initialize the class."""
         self.property_dict = property_dict
         self.property_keys = property_keys
         self.property_list = property_list
 
     def _translate_id(self, entity_id):
+        """Return entity identifier."""
```
Comment (peterjc) [inline Bio/PDB/AbstractPropertyMap.py]: Minor, but can you make this:

```python
        """Return entity identifier (PRIVATE)."""
```

File: Bio/PDB/Atom.py [hunk 2]
```diff
@@ -16,14 +16,17 @@
...
+
     def __init__(self, name, coord, bfactor, occupancy, altloc, fullname, serial_number,
                  element=None):
-        """Create Atom object.
-
-        The Atom object stores atom name (both with and without spaces),
-        coordinates, B factor, occupancy, alternative location specifier
-        and (optionally) anisotropic B factor and standard deviations of
-        B factor and positions.
+        """Initialize Atom object.
```
Comment (peterjc) [inline Bio/PDB/Atom.py]: Why did you remove the four lines of explanation here?

File: Bio/PDB/Dice.py [hunk 3]
```diff
@@ -54,6 +57,7 @@ def accept_residue(self, residue):
         return 0
 
     def accept_atom(self, atom):
+        """Verify is atoms are not Hydrogen."""
```
Comment (peterjc) [inline Bio/PDB/Dice.py]: Typo? is - if?

**Comment by peterjc** [comment]
Nice work - and you fixed a few old typos too. Hopefully Joao can give some feedback on some of the new docstrings, but this is already a useful improvement. Thank you.

Don't worry about the ``RuntimeError: generator raised StopIteration`` on AppVeyor Python 3.7, that has been fixed now on #1695. The TravisCI failure looked like a false positive, I've rerun that...

File: Bio/PDB/Atom.py (hunk 2 above)
Comment (svalqui) [reply Bio/PDB/Atom.py]: I moved it from __init__ to the class definition of the __init__, Atom; because the class Atom did not have a DocString, the __init__ is just below the class definition, and the content seem good to encompass the class. I am happy to put it back in the __init__ and leave the class DocString as a one-liner; or to have it on both, looks a bit redundant as the __init__ is just below; happy for suggestions,

Comment (peterjc) [reply Bio/PDB/Atom.py]: Oh yeah - I must have missed that - good choice.

**Comment by svalqui** [comment]
#1203

**Comment by peterjc** [comment]
@JoaoRodrigues, @etal, @lennax, @jgreener64  can you look at this with your knowledge of the PDB code? The docstring improvements look sensible to me.

**Comment by jgreener64** [comment]
From a quick scan looks fine to me.

**Comment by peterjc** [comment]
Merged, thank you!
